src/environments/environment.development.ts

```diff
@@ -24,19 +24,20 @@
 // `ng build --env=prod` then `environment.prod.ts` will be used instead.
 // The list of which env maps to which file can be found in `.angular-cli.json`.
 
-const commonIP = 'https://183.82.107.186:8080/';
-const tmIP = 'https://183.82.107.186:8080/';
-const schedulerIP = 'https://183.82.107.186:8080/';
+const commonIP = 'https://amritwprdev.piramalswasthya.org/';
+const tmIP = 'https://amritwprdev.piramalswasthya.org/';
+const schedulerIP = 'https://amritwprdev.piramalswasthya.org/';
 
 // Without API MAN Configuration
-const COMMON_API_OPEN = `${commonIP}commonapi-v1.0/`;
-const COMMON_API = `${commonIP}commonapi-v1.0/`;
-const SCHEDULER_API = `${schedulerIP}schedulerapi-v1.0/`;
-const TM_API = `${tmIP}tmapi-v1.0/`;
+const COMMON_API_OPEN = `${commonIP}commonapi-3.0.0/`;
+const COMMON_API = `${commonIP}commonapi-v3.0.0/`;
+const SCHEDULER_API = `${schedulerIP}schedulerapi-v3.0.0/`;
+const TM_API = `${tmIP}tmapi-v3.0.0/`;
+const sessionStorageEncKey = '<%= SESSION_STORAGE_ENC_KEY %>';
```

Comment:
@NKOKKILIGADDA87 please refer to how it's done in other places. 
This notation is only required in the ci template.
Please leave it empty in other env files.

Reply:
Yes, that's fine.
Only ci.template passes through the build script (check scripts folder) where the pattern is replaced by an environment variable set in CI.